### PR TITLE
2487 connect trd to api

### PR DESCRIPTION
**Pull request by cohnjesse**
## Description
Original Ticket - https://github.com/department-of-veterans-affairs/va.gov-team/issues/2487

As a Veteran using Rated Disabilities, I need to see the real data coming from within the VA to inform me of my status.

## Testing done 
unit tests already exist for these components

## Screenshots
NA

## Acceptance criteria
- [x] Change the redux code so it uses the number from FindRatingInfo
- [x] Update the endpoint the redux action is using and potentially update how errors are handled
- [x] Check unit test still passes
- [x] Data from the API is being displayed in the front end components

### CODE REVIEW

File: src/applications/personalization/rated-disabilities/reducers/total-disabilities.js [hunk 1]
```diff
@@ -16,16 +16,15 @@ export function totalRating(state = initialState, action) {
         ...state,
         loading: false,
         error: {
-          status: '500',
-          res: 'failed to load', // This changes when the backend gets wired up to the frontend.
+          code: action.response.errors.code,
+          detail: action.response.errors.detail,
```
Comment (micahchiang) [inline src/applications/personalization/rated-disabilities/reducers/total-disabilities.js]: Errors that get returned by an API call come back as an array. These values need to be extracted.
Screen Shot 2019-11-21 at 3 18 04 PM

**Review by micahchiang — CHANGES_REQUESTED** [review]
One change is needed for correct error states to be displayed in the `TotalRatedDisabilities` component.

File: src/applications/personalization/rated-disabilities/reducers/total-disabilities.js [hunk 2]
```diff
@@ -16,16 +16,15 @@ export function totalRating(state = initialState, action) {
...
+          detail: action.response.errors[0].detail,
         },
       };
     case FETCH_TOTAL_RATING_SUCCEEDED:
       return {
         ...state,
         loading: false,
-        error: null,
-        totalDisabilityRating: 80, // to be replaced with response payload
+        totalDisabilityRating: action.response.userPercentOfDisability,
```
Comment (micahchiang) [inline src/applications/personalization/rated-disabilities/reducers/total-disabilities.js]: Nice thing about finally getting this done is that we can use the same endpoint (hopefully) to check if a user is > 30% for the 686.

**Review by micahchiang — APPROVED** [review]
LGTM

**Review by U-DON — APPROVED** [review]

**Pull request merged**